pkg/db/migrations/9_canary_component_relationships.sql

```diff
@@ -0,0 +1,22 @@
+-- +goose Up
+-- +goose StatementBegin
+
+CREATE TABLE check_component_relationships(
```

Comment:
We need to add this table into the seed, and all the other changes
and then in the seed we can insert into canary_checker_version 4, so that it skips the migrations for new db's